app/src/main/res/layout/dialog_nearby.xml

```diff
@@ -11,8 +11,7 @@
   <TextView
```

Comment:
Did you find where this TextView appears on the main branch? If yes would you mind posting a screenshot?
If it is really never shown, better remove the whole TextView.

Reply:
Apologies oversight on my end, looks like there are no usages will delete.